ssylka ne ponravilas

kriskriskris0 opened the pull request:
request or not request this is the question

kriskriskris0 committed:
ssylka ne ponravilas

kazekagyeee commented on main.py:
```diff
@@ -2,9 +2,9 @@
 import pyautogui
 
-url = "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
+url = "https://turkish-tv-series.tv/turkey-serials/22792-klyukvennyy-scherbet7-vse-srii-smotret-tureckiiy-serial-na-russkom-yazyke4/99-seria.html"
```
верни давай как было

kriskriskris0 replied:
да чего опять мои сериалы отвергают

kazekagyeee replied:
+ у тебя конфликт в ветке, решай

kriskriskris0 committed:
?

kriskriskris0 committed:
Merge branch 'main' into 2nd

# Conflicts:
#	main.py

kriskriskris0 committed:
Загружает все CSV файлы из указанной папки и объединяет их в один DataFrame.
    :param folder_path: Путь к папке с CSV файлами.
    :return: Объединенный DataFrame.
